scalyr_agent/builtin_monitors/linux_process_metrics.py

```diff
@@ -834,19 +832,19 @@ def _initialize(self):
         self.__include_child_processes = self._config.get(
             'include_child_processes', default=False, convert_to=bool
         )
-        if not getattr(self, 'last_discovered', False):
+
+        self.__trackers = {}  # key -> process id, value -> ProcessTracker object
+        if not getattr(self, 'last_discovered', None):
```

Comment:
Just write this as `self.__last_discovered = None`.  No need to be checking this attribute now.